screens/index.js

```diff
@@ -34,6 +34,24 @@ const Index = () => {
   const [popularRestaurants, setPopularRestaurants] = useState({ stores: [] });
 
   const [foodTypeScreen, showFoodTypeScreen] = useState(false);
+  const foodTypesRef = useRef(null);
+  const searchBarRef = useRef(null);
```

Comment:
I tried doing something like this but if you click anywhere on the screen that is not the search bar, it freezes the screen and crashes the site

Reply:
it was something about it not being able to handle a null reference